presto-hive/src/test/java/io/prestosql/plugin/hive/metastore/cache/TestCachingHiveMetastore.java

```diff
@@ -261,15 +261,26 @@ public void testListRoles()
     public void testGetTableStatistics()
     {
         assertEquals(mockClient.getAccessCount(), 0);
-        assertEquals(metastore.getTableStatistics(IDENTITY, TEST_DATABASE, TEST_TABLE), TEST_STATS);
+
+        Table table = metastore.getTable(IDENTITY, TEST_DATABASE, TEST_TABLE).get();
+        assertEquals(mockClient.getAccessCount(), 1);
+
+        assertEquals(metastore.getTableStatistics(IDENTITY, table), TEST_STATS);
```

Comment:
The test doesn't show improvement, since it still ends with `assertEquals(mockClient.getAccessCount(), 2);`

Please add
```
assertEquals(mockClient.getAccessCount(), 0);
Table table = metastore.getTable(IDENTITY, TEST_DATABASE, TEST_TABLE).get();
assertEquals(mockClient.getAccessCount(), 1);
assertEquals(metastore.getTableStatistics(IDENTITY, table), TEST_STATS);
assertEquals(mockClient.getAccessCount(), 3);    <--- 3 not 2
```

in the commmit which adds this commit initially


same for the other test